examples/milp_factory_schedule.jl

```diff
@@ -0,0 +1,190 @@
+#  Copyright 2017, Iain Dunning, Joey Huchette, Miles Lubin, and contributors
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#############################################################################
+# JuMP
+# An algebraic modeling language for Julia
+# See http://github.com/JuliaOpt/JuMP.jl
+#############################################################################
+
+"""
+      example_factory_schedule()
```

Comment:
This docstring is in the wrong place. It should be right before the definition of `example_factory_schedule`. See https://docs.julialang.org/en/v1/manual/documentation/index.html#Documentation-1.

Reply:
corrected in latest commit, that seems to have also fixed the coverage CI warning